### PR TITLE
Feat/vsdspub 471/import connectors

### PR DESCRIPTION
**Pull request by Yalz**

### CODE REVIEW

File: ldi-orchestrator/ldio-connectors/ldio-console-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioConsoleOutAutoConfig.java [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 	}
 
 	public static class LdioConsoleOutConfigurator implements LdioConfigurator {
 		@Override
-		public LdiComponent configure(Map<String, String> properties) {
-			return new LdiConsoleOut(properties);
+		public LdiComponent configure(ComponentProperties properties) {
+			Lang outputLanguage = properties.getOptionalProperty("content-type")
+					.map(contentType -> getLang(MediaType.valueOf(contentType)))
+					.orElse(Lang.NQUADS);
```
Comment (pj-cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-console-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioConsoleOutAutoConfig.java:30]: Would it be useful to make this a constant?

File: ldi-orchestrator/ldio-connectors/ldio-http-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioHttpOutAutoConfig.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			RestTemplate restTemplate = new RestTemplate();
+			HttpHeaders headers = new HttpHeaders();
+
+			Lang outputLanguage = config.getOptionalProperty("content-type")
+					.map(contentType -> {
+						Lang lang = getLang(Objects.requireNonNull(MediaType.valueOf(contentType)));
+						headers.setContentType(MediaType.valueOf(lang.getContentType().getContentTypeStr()));
+						return lang;
+					})
+					.orElse(Lang.NQUADS);
```
Comment (pj-cegeka) [inline ldi-orchestrator/ldio-connectors/ldio-http-out/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioHttpOutAutoConfig.java:42]: Could also be same constant?

**Review by pj-cegeka — APPROVED** [review]

File: ldi-core/ldes-client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+        <artifactId>ldi-core</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ldes-client</artifactId>
+
+    <properties>
+        <formatter.config-file>${project.basedir}/../formatter.xml</formatter.config-file>
```
Comment (Tomvbe) [inline ldi-core/ldes-client/pom.xml:13]: another formatter leftover?

File: ldi-core/ldes-client/pom.xml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-annotations</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.1.1</version>
```
Comment (Tomvbe) [inline ldi-core/ldes-client/pom.xml:83]: Can we use properties for the versions in this pom? I'm guessing that we use mockito-core in other modules as well.

File: ldi-core/version-materialiser/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionMaterialiser.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+		this.restrictToMembers = restrictToMembers;
+	}
+
+	@Override
+	public Model transform(Model linkedDataModel) {
+		Model versionMaterialisedModel = ModelFactory.createDefaultModel();
+
+		Map<Resource, Resource> versionIDEntityIDMap = getVersionIDEntityIDMap(linkedDataModel, versionPredicate);
+
+		linkedDataModel.listStatements().forEach(statement -> {
```
Comment (Tomvbe) [inline ldi-core/version-materialiser/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/VersionMaterialiser.java:29]: I find this method a bit difficult to read when you define methods at the top and then change some on the way to finally use them at the end. I feel that it is more maintainable when you make them immutable. Maybe split them in separate methods?

**Review by Tomvbe — APPROVED** [review]

File: ldi-core/ldes-client/pom.xml (hunk 4 above)
Comment (Yalz) [reply ldi-core/ldes-client/pom.xml:83]: will be fixed in final iteration of story

**Pull request merged**